app/main.py

```diff
@@ -1,13 +1,23 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
     pass
 
 
-def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
+def format_single_linter_file(file_path: str, errors: list):
+    return {"errors": [format_linter_error(error) for error in errors],
+            "path": file_path,
+            "status": ''.join(["failed" if len(errors) != 0 else "passed"])
+            }
```

Comment:
Bad code style, you have alone closing curly brace and not alone the opening one. Either make keys starting on a new line or move the closing brace to the preceding line.